lib/matplotlib/axes/_axes.py

```diff
@@ -3525,6 +3525,10 @@ def boxplot(self, x, notch=None, sym=None, vert=None, whis=None,
 
           - ``means``: points or lines representing the means.
 
+        Notes
+        -----
+        .. [Notes section required for data comment. See #10189.]
```

Comment:
I'm guessing this is the source of the problem.

Reply:
No, it was an indented `%(FancyArrow)s`. For now, I've unindented it again, which messes up the Other Parameters section, but at least builds. Might be fixed with #10364.